### PR TITLE
chore: Sort features

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Just runs a sort on these lines

### CODE REVIEW

**Review by wmak — APPROVED** [review]
🤔 is there a way to lint this so we're not doing this every n months?

File: src/sentry/features/__init__.py [hunk 1]
```diff
@@ -178,16 +181,16 @@
...
 default_manager.add("organizations:integrations-custom-scm", OrganizationFeature)
-default_manager.add("organizations:integrations-feature-flag-integration", OrganizationFeature)
 default_manager.add("organizations:integrations-event-hooks", OrganizationFeature)
+default_manager.add("organizations:integrations-feature-flag-integration", OrganizationFeature)
 default_manager.add("organizations:integrations-incident-management", OrganizationFeature)
 default_manager.add("organizations:integrations-issue-basic", OrganizationFeature)
 default_manager.add("organizations:integrations-issue-sync", OrganizationFeature)
 default_manager.add("organizations:integrations-stacktrace-link", OrganizationFeature)
 default_manager.add("organizations:integrations-ticket-rules", OrganizationFeature)
 default_manager.add("organizations:performance-issue-details-backend", OrganizationFeature)
```
Comment (ceorourke) [inline src/sentry/features/__init__.py:191]: Can you scooch this up out of the "NOTE: Don't add features down here!" section while you're at it?

Comment (evanpurkhiser) [reply src/sentry/features/__init__.py:191]: where should it go? just up a single line?

Comment (evanpurkhiser) [reply src/sentry/features/__init__.py:181]: ohhh I see

Comment (evanpurkhiser) [reply src/sentry/features/__init__.py:181]: I'll do in a diff PR actually

**Review by asottile-sentry — APPROVED** [review]

**Pull request merged**